clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackLoadBalancerProvider.groovy

```diff
@@ -190,6 +191,43 @@ interface OpenstackLoadBalancerProvider {
   ActionResponse deleteMonitor(final String region, final String id)
 
   /**
+   * Get port from load balancer listener description. Openstack load balancers have no native concept of internal port,
+   * so we store in the description field of the load balancer.
+   *
+   * This may be changed in a future version.
+   *
+   * @param pool
+   * @return
+   */
+  Integer getInternalLoadBalancerPort(String region, String listenerId)
```

Comment:
That `@param` lists don't match the method signature for each of these 4 methods.
I'd just drop the `@param` lists entirely (and the `@return`'s) if they're not going to include anything beyond the name of the parameters.